borsh-ts/test/serialize.test.js

```diff
@@ -91,9 +91,27 @@ test('serialize/deserialize with class methods', () => {
     expect(newValue).toEqual(item);
 });
 
+test('serialize/deserialize fixed array', () => {
+    const value = new Test({
+        a: ['hello', 'world']
+    });
+    const schema = new Map([[Test, {
+        kind: 'struct',
+        fields: [
+            ['a', ['string', 2]]
+        ]
+    }]]);
+
+    const buf = borsh.serialize(schema, value);
+    const deserializedValue = borsh.deserialize(schema, Test, buf);
+    
+    expect(buf).toEqual(Buffer.from([5, 0, 0, 0, 104, 101, 108, 108, 111, 5, 0, 0, 0, 119, 111, 114, 108, 100]));
+    expect(deserializedValue.a).toEqual(['hello', 'world']);
+});
+
```

Comment:
This test looks nice. I think we also need a test to test error to serilaize/deserialize when given array has a different array in schema.